doc/en/api/1.0.0/gwclayers.yaml

```diff
@@ -0,0 +1,236 @@
+---
+swagger: '2.0'
+info:
+  version: 1.0.0
+  title: GeoWebCache Layers
+  description: A layer is a published resource (feature type or coverage).
+schemes:
+- http
+host: localhost:8080
+basePath: /geoserver/gwc/rest
+paths:
+  /layers:
+    get:
+      operationId: layersGet
+      summary: Get a list of cached layers
+      description: Displays a list of all cached layers on the server. You must use the "Accept:" header to specify format or append an extension to the endpoint (example "/layers.xml" for XML)
+      produces:
+        - application/xml
+      responses:
+        200:
+          description: OK
+          schema:
+            $ref: "#/definitions/Layers"
+          examples:
+            application/xml: |
+              <layers>
+                <layer>
+                  <name>opengeo:countries</name>
+                  <atom:link xmlns:atom="http://www.w3.org/2005/Atom" rel="alternate" href="http://localhost:8080/geoserver/gwc/layers/opengeo%3Acountries.xml" type="text/xml"/>
+                </layer>
+                <layer>
+                  <name>usa:states</name>
+                  <atom:link xmlns:atom="http://www.w3.org/2005/Atom" rel="alternate" href="http://localhost:8080/geoserver/gwc/layers/usa%3Astates.xml" type="text/xml"/>
+                </layer>
+              </layers>
+
+  /layers/{layerName}:
+    get:
+      operationId: layersNameGet
+      summary: Retrieve a cached layer
+      description: Retrieves a single cached layer definition. Use the "Accept:" header to specify format or append an extension to the endpoint (example "/layers/{layer}.xml" for XML).
+      produces:
+        - application/xml
+      parameters:
+        - name: layerName
+          in: path
+          required: true
+          description: The name of the layer to retrieve.
+          type: string
+        - name: quietOnNotFound
+          in: query
```

Comment:
We've been ignoring / removing the quietOnNotFound parameter everywhere, since it technically applies to all endpoints. I'll be adding a note to the top-level api docs, but that is about it.

See discussion here: https://github.com/geoserver/geoserver/pull/2530